### PR TITLE
P1 lists typo rating

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Ratings, Typo, Lists., Slider done (Responsive)

To do:
Start working on the next P1 components

Bugs:
Sidebar:
If the user tests responsive then the sidebar elements get invisible.
aside needs to be given static height (for now set to 500vh)

### CODE REVIEW

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import "../design-guide.css";
+
+* {
+  box-sizing: border-box;
+}
+
+h1#h1 {
```
Comment (deekshithmd) [inline components/typography/typography.css:7]: using class will be good instead of id

**Review by deekshithmd — APPROVED** [review]

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: components/typography/typography.css (hunk 1 above)
Comment (mayuresh-tech) [reply components/typography/typography.css:7]: It was conflicting with my documentation page specificity so I used ID. 
For now, I will keep it like this but will think about how to remove it.

File: components/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+@import "../design-guide.css";
+
+* {
+  box-sizing: border-box;
+}
```
Comment (harshkotkar5557) [inline components/lists/lists.css:5]: You can avoid to use in every CSS file by simply add it in main file.

**Review by harshkotkar5557 — COMMENTED** [review]
looks good to me, though I have commented

File: components/lists/lists.css (hunk 2 above)
Comment (mayuresh-tech) [reply components/lists/lists.css:5]: Done
Fixed in later commits

**Pull request merged**